src/com/serotonin/mango/rt/dataImage/PointValueCache.java

```diff
@@ -36,14 +37,19 @@
     private final int dataPointId;
     private final int defaultSize;
     private final PointValueDao dao;
+    private int maxSize = 0;
```

Comment:
Are these changes absolutely necessary in order to add information in the annotations, the change in this class seems to me to require a separate big task. It seems to me that your code turns off the cash.

Reply:
Grzegorz if you think that maxSize is a new property in this class, please look on previous one version body of this class.This value is very important in this class and it was. I do not know what I should write here. For me this looks like, you se something new in class but can you next time try compare with code from main branche. In this case siv/develop.